### PR TITLE
Neuron logo folder added / mix.exs updated / Readme updated

### PR DESCRIPTION
**Pull request by reallinfo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<p align="center"><img src="logo/horizontal.png" alt="neuron" height="150px"></p>
+
 # Neuron
```
Comment (uesteibar) [inline README.md]: You can remove this line now that we have a fancy logo 🎉

File: mix.exs [hunk 2]
```diff
@@ -54,6 +54,7 @@ defmodule Neuron.Mixfile do
     [
       source_ref: "v#{@version}",
       main: "Neuron",
+logo: "neuron/logo/logo.png",
```
Comment (uesteibar) [inline mix.exs]: Could you move this to the same level of indentation as the rest of the items in this list?

Also, I believe the correct path here is `logo/logo.png`, without `neuron/`.

**Review by uesteibar — COMMENTED** [review]
Thanks for this!

I left some comments on things that would need to be changed in order for this to be merged.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,3 @@
-<p align="center"><img src="logo/horizontal.png" alt="neuron" height="150px"></p>
```
Comment (uesteibar) [inline README.md]: I wasn't referring to removing this line but the one below, the one that says `# Neuron`. Sorry if I didn't explain myself correctly.

**Review by uesteibar — APPROVED** [review]
This looks great, thank you! ❤️

**Comment by reallinfo** [comment]
You're welcome. feel free to say if you have another request. Btw, I'm sorry for the confusion :) Soon I'll add also editable files via comment.

**Comment by reallinfo** [comment]
https://drive.google.com/drive/folders/1er8lfMvl5ubVk4Y1O7Zrjr4sJo1o6qF6

here are all the files. Have a nice day!

**Pull request merged**